tests/e2e/e2e_test.go

```diff
@@ -51,11 +51,11 @@ var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
 	subnets := vms.XSVMSubnetsOrPanic(nodes...)
 
 	upgrades := upgrade.Default
-	if flagVars.ActivateGranite() {
-		upgrades.GraniteTime = upgrade.InitiallyActiveTime
-		upgrades.GraniteEpochDuration = 4 * time.Second
+	if flagVars.ActivateLatest() {
+		// Activate all upgrades up to and including the latest
+		upgradetest.SetTimesTo(&upgrades, upgradetest.Latest, upgrade.InitiallyActiveTime)
```

Comment:
Is this acceptable? This is functional change, as it will set all prior upgrade times, rather than just the specific upgrade.

Reply:
because all the prior upgrade times are already set, correct?